pyface/tasks/tests/test_action_manager_builder.py

```diff
@@ -8,6 +8,7 @@
 #
 # Thanks for using Enthought open source!
 
+from contextlib import contextmanager
 import unittest
 
 
```

Comment:
imports from pyface.tasks.action.api below mayb should get updated to use new pyface.actions.schema.api

Reply:
Yep - missed this.  There are more places that the imports need to be changed (eg. examples, docs).